schema.proto

```diff
@@ -0,0 +1,277 @@
+syntax = "proto2";
+
+package biscuit.format.schema;
+
+message Biscuit {
+  optional uint32 rootKeyId = 1;
+  required SignedBlock authority = 2;
+  repeated SignedBlock blocks = 3;
+  required Proof proof = 4;
+}
+
+message SignedBlock {
+  required bytes block = 1;
+  required PublicKey nextKey = 2;
+  required bytes signature = 3;
+  optional ExternalSignature externalSignature = 4;
+  optional uint32 version = 5;
+}
+
+message ExternalSignature {
+  required bytes signature = 1;
+  required PublicKey publicKey = 2;
+}
+
+message PublicKey {
+  required Algorithm algorithm = 1;
+
+  enum Algorithm {
+    Ed25519 = 0;
+    SECP256R1 = 1;
+  }
+
+  required bytes key = 2;
+}
+
+message Proof {
+  oneof Content {
+    bytes nextSecret = 1;
+    bytes finalSignature = 2;
+  }
+}
+
+message Block {
+  repeated string symbols = 1;
+  optional string context = 2;
+  optional uint32 version = 3;
+  repeated FactV2 facts_v2 = 4;
+  repeated RuleV2 rules_v2 = 5;
+  repeated CheckV2 checks_v2 = 6;
+  repeated Scope scope = 7;
+  repeated PublicKey publicKeys = 8;
+}
+
+message Scope {
+  enum ScopeType {
+    Authority = 0;
+    Previous  = 1;
+  }
+
+  oneof Content {
+    ScopeType scopeType = 1;
+    int64 publicKey = 2;
+  }
+}
+
+message FactV2 {
```

Comment:
the protobuf schema has been updated in the spec repo. It does not change anything in the format itself, but some fields and types have been renamed to remove the `V2`s where it makes sense.